test/gtest/uct/test_event.cc

```diff
@@ -166,20 +170,38 @@ void test_uct_event::test_recv_am(unsigned arm_flags, unsigned send_flags)
     free(recv_buffer);
 }
 
-UCS_TEST_SKIP_COND_P(test_uct_event, am,
+/*
+ * Disable flow control (if transport supports it), and expect at most 2%
+ * spurious events.
```

Comment:
what are those 2%?

Reply:
removed the comment, not relevant anymore